scheduler/reconcile_test.go

```diff
@@ -1063,6 +1103,126 @@ func TestReconciler_LostNode_PreventRescheduleOnLost(t *testing.T) {
 	}
 }
 
+func TestReconciler_InformationalAllocs(t *testing.T) {
+	disabledReschedulePolicy := &structs.ReschedulePolicy{
+		Attempts:  0,
+		Unlimited: false,
+	}
+
+	ci.Parallel(t)
+	now := time.Now()
+
+	testCases := []struct {
+		name             string
+		count            int
+		stoppedCount     int
+		failedCount      int
+		reschedulePolicy *structs.ReschedulePolicy
+		expectPlace      int
+		expectStop       int
+		expectIgnore     int
+	}{
+		{
+			name:             "Count 3, 2 allocs failed, 1 stopped, no reschedule",
+			count:            3,
+			stoppedCount:     1,
+			failedCount:      2,
+			reschedulePolicy: disabledReschedulePolicy,
+			expectPlace:      2,
+			expectStop:       1,
+			expectIgnore:     1,
+		},
```

Comment:
This case appears to cover your comment @tgross, does it not? The desired behavior, if I'm correct, should be 2 placed allocs, 1 stopped and 1 ignored in this case, which is exactly what we're getting.